docker/bootstrap-conda.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Wondering whether this belongs in the docker folder since it is also needed for a local dev environment.